### ISSUE TITLE
Cannot stream to my LG WebOs 3.0 TV

### ISSUE THREAD

**Issue by rudesome** [popcorntime/popcorntime#508]
Operating System Version:
Windows 10

Popcorn Time Version:
0.3.10 Beta

Download date:
03-12-2017

#### Expected Behaviour
Stream to my LG WebOS 3.0 TV
...

#### Actual Behaviour
Since yesterday i wont get the icon for streaming to my TV
...

#### Steps to reproduce the behaviour
Open any movie or serie, next to watch now you get the option of selecting where you want to play it in. Normally my TV is also an option. I still can stream directly from my laptop to my TV with the wireless display adapter of Windows. I checked the upnp-mediarenderer folder but it isn't changed, only thing that has changed is that i installed a new update on my TV version 4.31.10 (Europe) but i did it Thursday and after the update it worked until yesterday.

**Comment by streger** [comment]
Well, it seems that your TV provider decided to bork you...
They are now probably using some kind of exclusions for custom ports that pt uses.

**Comment by rudesome** [comment]
I have tried some things:

1. Thought it could be something in my home router so i bought a new one and installed Openwrt on it. 
2. Different versions of popcorn.
3. replaced: upnp-mediarenderer-client module with an older version in the popcorn folder.

Sometimes i see my TV in popcorn, especially after a fresh install of popcorn and after a reboot of my router... When selecting my TV in popcorn and the stream starts after a few minutes i get the error message on my TV: "instable network" and the stream stops.

I found a workaround and it works with no problems.. Start the movie in butter and connect my PC with the TV in Windows 10, no issues... So it is possible over my network but not with the direct stream in popcorn. The only downside of it is, that i cannot control the sync of the subtitles.

I see that there is a new version of popcorn out: Popcorn Time 4.1.5. Gonna try that one this weekend, will update, this post, thanks for responding Streger.
